packages/client/src/components/Cards/Deck.tsx

```diff
@@ -0,0 +1,19 @@
+import { theme } from 'theme';
+import { Box, BoxProps } from 'theme-ui';
+import back from 'images/back.jpg';
+import { baseCard } from './style';
+
+export const Deck = ({ cardLeft, ...rest }: { cardLeft: number } & BoxProps) => {
```

Comment:
`cardLeft` -> `cardsNumber`, `remainingCardsNumber`

Reply:
gotcha, I thought cardLeft was a better name ;-p

Reply:
The other way would be to just have `cardsNumber`/`cardsCount` and maybe a `maxCardsNumber`/`maxCardsCount`. I actually just remembered I try to use `count` for these kind of variables.